TESTING.md

```diff
@@ -470,6 +471,40 @@ To run these tests, you go inside `test-artifacts` and execute:
 
 The execution then proceeds as explained in [Binary testing](#Binary-testing).
 
+## Conditional running of tests
+
+The whole test suite takes several hours to run. If some errors happen during the run, we need to clean up and try again
+from the beginning, which is not always convenient.
+There are a few tags that help us gain some control on the flow:
+
+* `-vcd-pre-post-checks`    Perform checks before and after tests (false)
```

Comment:
Please add some words and/or formatting describing what kind of "checks" - to emphasize this flag, so it's clear for a new user that this is a global setting for the whole section (if I got it right?).